src/cryptography/x509/name.py

```diff
@@ -29,7 +29,7 @@ class _ASN1Type(utils.Enum):
 
 _ASN1_TYPE_TO_ENUM = {i.value: i for i in _ASN1Type}
 _SENTINEL = object()
```

Comment:
You should remove this if we're not going to explicitly use the sentinel any more

Reply:
done